dev/io.openliberty.microprofile.reactive.streams.operators30.internal/bnd.bnd

```diff
@@ -0,0 +1,83 @@
+#*******************************************************************************
+# Copyright (c) 2019, 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props,\
+          ~../cnf/resources/bnd/app-resources.bnd
+
+bSymbolicName=io.openliberty.microprofile.reactive.streams.operators30.internal
+bVersion=1.0
+
+
+Bundle-Name: io.openliberty.microprofile.reactive.streams.operators30.internal
+Bundle-SymbolicName: io.openliberty.microprofile.reactive.streams.operators30.internal
+
+Bundle-Description:MicroProfile Reactive Streams 3.0 impl, version ${bVersion}
+
+WS-TraceGroup: REACTIVESTREAMS
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+-dsannotations-inherit: true
+-dsannotations: io.openliberty.microprofile.reactive.streams.operators30.spi.impl.LibertyReactiveStreamsEngineImpl
+
+Import-Package: \
+  !io.openliberty.microprofile.reactive.streams.operators.spi.impl, \
+  org.eclipse.microprofile.reactive.streams.operators.core; version="[1.0,1.1)", \
+  org.eclipse.microprofile.reactive.streams.operators.spi; version="[1.0,1.1)", \
+  org.eclipse.microprofile.reactive.streams.operators; version="[1.0,1.1)", \
+  *
+
+# See the app-resources.bnd included above
+app-resources= \
+  META-INF/services/org.eclipse.microprofile.reactive.streams.operators.spi.ReactiveStreamsEngine
+
+Export-Package: io.openliberty.microprofile.reactive.streams.operators30.spi.impl;thread-context=true;version='1.0'
```

Comment:
Do we still need `thread-context=true` here?

Reply:
don't know